ppafm/ml/Generator.py

```diff
@@ -389,29 +395,33 @@ def __init__(
         self.df_steps = self.afmulator.df_steps
         self.z_size = self.scan_dim[2] - self.df_steps + 1
 
-    def _prepareBuffers(self, rhos=None, rho_deltas=None):
+    def _prepare_tip_buffers(self, rhos=None, rho_deltas=None, density_cutoff=None):
```

Comment:
Jush checking - is the KPFM procedure using buffers as well @mondracek? Do you have an idea about it? Because if so, then restriction to Tip is maybe not what you want?

Reply:
This is about the GPU buffers. I don't think the KPFM has been implemented in the GPU version yet.